lang/jmh/org/partiql/jmh/PartiQLBenchmark.kt

```diff
@@ -91,4 +90,4 @@ open class PartiQLBenchmark {
         val exprValue = state.expression.eval(state.session)
         blackhole.consume(exprValue)
     }
-}
```

Comment:
Not sure what's going on here but for some odd reason ktlint decided it no longer liked the lack of newline here in my local build.  Not sure why it was ok with it previously...